server/controller/socialMediaController.js

```diff
@@ -0,0 +1,54 @@
+import Authenticate from '../middleware/auth/Authenticate';
+import models from '../models';
+
+const { User } = models;
+const socialMediaCallback = (accessToken, refreshToken, profile, done) => {
+  try {
+    const {
+      id, name, emails,
+    } = profile;
+
+    if (!emails) {
+      const noEmailError = 'No email was found in your account';
+      return done(null, noEmailError);
+    }
+
+    const email = emails[0].value;
+    User.findOrCreate({
```

Comment:
Hey, try rewriting this using async await, as that is the convention being used.
Using `.then` will be altering our agreed upon convention.

What error exactly do you get when you use `async await`? can you atleast show the error message together with your implementation.

Reply:
I have rewritten this... had to delete my cached and re-install all dependencies.